app/views/layouts/_navigation.html.haml

```diff
@@ -1,4 +1,4 @@
-%nav.govuk-header__navigation{"aria-label" => t('layouts.application.header.navigation.top_navigation')}
+%nav.govuk-header__navigation{"aria-label" => t('layouts.application.header.navigation.top_navigation'),role:"navigation"}
```

Comment:
Nits:
- add spacing after the comma
- add spacing after the colon

Coding standard: you're using both hash rocket `attribute => value` and json `attribute: value` syntax style, choose one

Reply:
Noted, this inconsistency is throughout the app, affecting 11 HAML files - will update this instance but also might be a sign that we need to review our HAML linting (or implement it) - making a ticket for this this morning